src/stream/src/common/table/state_table.rs

```diff
@@ -630,18 +637,24 @@ impl<S: StateStore> StateTable<S> {
         self.epoch = Some(new_epoch);
     }
 
-    pub async fn commit(&mut self, new_epoch: EpochPair) -> StreamExecutorResult<()> {
+    pub async fn commit(
+        &mut self,
+        new_epoch: EpochPair,
+        watermark: Option<&ScalarImpl>,
```

Comment:
I suggest an independent interface to pass this parameter and update `last_watermark`.

Reply:
> I suggest an independent interface to pass this parameter and update `last_watermark`.

fixed